exercicios/para-casa/entregas/artemiza-rocha/resolucao2.js

```diff
@@ -0,0 +1,30 @@
+/*Escreva uma classe chamada InvoiceItem,
+que modela um item de uma compra, com id, description, quantity e price. */
+
+// criar dois metodos
+// criar metodo preço unitário * quantidade
+
+class InvoiceItem {
+    constructor(id, description, quantity, price ){
+        this.id = id;
+        this.description = description;
+        this.quantity = quantity;
+        this.price = price;
+    }
+
+    getProdId(){
```

Comment:
Oie! Tudo bem? =] Fiquei intrigada no que diz respeito a implementação do método. Neste caso do getProdId, tenciona-se retornar o Id do produto, correto? No entanto está retornando a descrição.  Sugiro alteração para que retorne o id conforme nome do método substituindo o this.description. Qualquer dúvida basta entrar em contato comigo que terei prazer em auxiliar!